### PR TITLE
feat: introduce eXo parent pom - EXO-64103

### PR DESCRIPTION
**Pull request by rdenarie**
This feature introduce new parent pom for eXo to be able to declare libraries versions used only in eXo, without impacting meeds

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,1770 +22,41 @@
...
-    <org.codehaus.cargo.version>0.9</org.codehaus.cargo.version>
-    <org.codehaus.groovy.version>2.4.12</org.codehaus.groovy.version>
-    <org.cometd.version>6.0.7</org.cometd.version>
-    <!-- jetty-client is a dependency coming from cometd. as we use cometd 6.0.7, the version for jetty is 10.0.11,
-    we need to define this version to use dependency jetty-util-ajax, which is optionnal in cometd
-    -->
-    <version.eclipse.jetty-client>10.0.11</version.eclipse.jetty-client>
+    <org.exoplatform.social.version>6.5.x-security-fix-SNAPSHOT</org.exoplatform.social.version>
+    <addon.exo.tasks.version>3.5.x-security-fix-SNAPSHOT</addon.exo.tasks.version>
+    <addon.exo.notes.version>1.3.x-security-fix-SNAPSHOT</addon.exo.notes.version>
```
Comment (azayati) [inline pom.xml]: - The versions are not good

File: pom.xml [hunk 2]
```diff
@@ -22,1770 +22,41 @@
...
-        <groupId>ch.qos.logback</groupId>
-        <artifactId>logback-classic</artifactId>
-        <version>${ch.qos.logback.version}</version>
+        <groupId>org.exoplatform.social</groupId>
+        <artifactId>social</artifactId>
+        <version>${org.exoplatform.social.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
       </dependency>
       <dependency>
```
Comment (azayati) [inline pom.xml:47]: this deleted configuration should be inherited from maven-depmgt-exo-pom ?

**Review by azayati — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rdenarie) [reply pom.xml]: modified

File: pom.xml (hunk 2 above)
Comment (rdenarie) [reply pom.xml:47]: No, when creating the module, ITOP copy the pom from maven-depmgt-pom as is.
As dependencies are already declared in maven-depmgt-pom, we dont need it here.

We didnt evaluate which dep should be in maven-exo-depmgt-pom or in maven-depmgt-pom, for the moment we only create the module structure.
When all is good for that, we will work on moving deps declaration

**Review by azayati — APPROVED** [review]
LGTM

**Review by ahamdi — APPROVED** [review]

**Pull request merged**